### PR TITLE
docs: update macOS installation instructions for Bazel

### PR DESCRIPTION
**Pull request by creachadair**
Rather than give users manual installation and upgrade instructions for Bazel,
use the Bazelisk command-line tool, which knows how to use the metadata in
the repository to pick the correct version and to download/install it.

### CODE REVIEW

File: kythe/web/site/getting-started-macos.md [hunk 1]
```diff
@@ -66,11 +66,17 @@ done
...
+# Since July 2020, Bazel is now part of Homebrew Core, and can therefore
+# be installed as any ordinary Homebrew package. Note, however, that
+# version updates in Homebrew may be out of sync with Kythe versions, so
+# we recommend you manually pin the current version after installing.
+#
+# When the Kythe .bazelversion file updates, you can upgrade Bazel thus:
+#
+#   brew unpin bazel ; brew upgrade bazel && brew pin bazel
+#
+brew install bazel
```
Comment (salguarnieri) [inline kythe/web/site/getting-started-macos.md:78]: Is there a reason to not use [bazelisk](https://github.com/bazelbuild/bazelisk) so the user doesn't have to manually manage bazel upgrades?

Comment (creachadair) [reply kythe/web/site/getting-started-macos.md:78]: I don't know, I've never used it. I will take a look and report back. :)

Comment (creachadair) [reply kythe/web/site/getting-started-macos.md:78]: It looks like `bazelisk` works fine—at least to the point where I can build and run tests. I'll update this change to point there instead.

Comment (salguarnieri) [reply kythe/web/site/getting-started-macos.md:78]: great, thanks!

Comment (creachadair) [reply kythe/web/site/getting-started-macos.md]: I updated the text, PTAL.

**Review by salguarnieri — APPROVED** [review]
thanks

**Comment by creachadair** [comment]
Before you merge this, let me rewrite the description.

**Comment by creachadair** [comment]
> Before you merge this, let me rewrite the description.

Ok, good to go. If you prefer I can squash the commits myself.

**Comment by salguarnieri** [comment]
> > Before you merge this, let me rewrite the description.
> 
> Ok, good to go. If you prefer I can squash the commits myself.

I updated the branch to do the squash and merge but our buildbot seems stuck. Can you squash the commits to see if buildbot picks up the change?

**Comment by creachadair** [comment]
> I updated the branch to do the squash and merge but our buildbot seems stuck. Can you squash the commits to see if buildbot picks up the change?

Done. I reordered the commits to put the merge first, and forced update on this branch.

**Comment by creachadair** [comment]
You may need to tag this PR with "buildbot" since I am no longer a real person as far as the automation is concerned.

**Comment by salguarnieri** [comment]
> You may need to tag this PR with "buildbot" since I am no longer a real person as far as the automation is concerned.

Thanks :)

**Pull request merged**